nested/vmsetupparams.json

```diff
@@ -81,7 +81,8 @@
                 "adminPassword": "[parameters('moodleCommon').dbLoginPassword]",
                 "mssqlDbServiceObjectiveName": "[parameters('moodleCommon').mssqlDbServiceObjectiveName]",
                 "mssqlDbEdition": "[parameters('moodleCommon').mssqlDbEdition]",
-                "mssqlDbSize": "[parameters('moodleCommon').mssqlDbSize]"
+                "mssqlDbSize": "[parameters('moodleCommon').mssqlDbSize]",
```

Comment:
LGTM. MySQL Version added to VMSS template. Good to be used with helper functions script.